README.md

```diff
@@ -95,6 +95,33 @@ because they need additional setup on your system:
 
       pip install slycot
 
+  Note that building Slycot might fail for the following reasons:
+  * The Slycot package contains a cmake check which fails when it
```

Comment:
Perhaps there should be an empty line above the start of a list.

Reply:
Done.